src/desktop/apps/artsy-v2/server.tsx

```diff
@@ -34,6 +35,7 @@ const allRoutes = flatten(
  * Mount non-Reaction routes that are relevant to specific global router routes
  */
 app.get("/artwork/:artworkID/download/:filename", handleArtworkImageDownload)
+app.get("/collection/:collectionSlug", handleCollectionToArtistSeriesRedirect)
```

Comment:
Is there a reason to pull this out vs. doing it in the style of `searchMiddleware` and `artistMiddleware` below?

Reply:
That would get hit on every page supported by the new router, whereas this functionality just for the collection route. 

Those routes all `next()` if they're not on the artist page (for example), so those _could_ be rewritten in this style too. This way feels a little better/more targeted.